Dockerfile

```diff
@@ -63,5 +63,7 @@ ENV PATH $PATH:$GOPATH/bin
 # eclair
 RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
 
+VOLUME /root/lightning-integration/output
```

Comment:
Probably also a good idea to declare `results/` a volume, since that's where the raw `py.test` output is stored and it might help debugging a bunch if we keep it.

Reply:
Did you mean `reports/`?
I added it.